themes/bootstrap3/templates/epf/search.phtml

```diff
@@ -0,0 +1,2 @@
+<?php
+  echo $this->render('search/results.phtml');
```

Comment:
Can be simplified as above.

Reply:
This is now a two-line file, so I think the <?= simplification is no longer useful.

Reply:
Agreed.